Integrations/integration-ArcherRSA.yml

```diff
@@ -2611,7 +2611,7 @@ script:
             fieldsToValue = JSON.parse(dataArgs.fieldsToValues);
         }
         var incidentId = dataArgs.incidentId;
-        if (incidentId){
+        if (incidentId.toString()){
             contentId = getContentIdsFromIncIds(appId, incidentId)[0];
```

Comment:
```suggestion
        if (incidentId) {
            contentId = getContentIdsFromIncIds(appId, incidentId.toString())[0];
```

Reply:
you also have it in `archer-get-record`

Reply:
I just noticed I only covered the places where the incidentId received from dataArgs and not from args. I'd do a quick fix